### PR TITLE
Add toggle password visibility button on login form

### PR DESCRIPTION
**Pull request by flo0627**
Hello,

Changes proposed in this pull request:

- Add toggle password visibility button on login form

Capture d’écran du 2020-10-02 17-15-25

How to test the feature manually:

1. On the login form test the new button,
2. Repeat 1. for each themes.

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested
- [ ] unit tests written (optional if too hard)
- [ ] documentation updated

### CODE REVIEW

File: app/views/auth/formLogin.phtml [hunk 1]
```diff
@@ -15,7 +15,10 @@
 
 		<div class="form-group">
 			<label for="passwordPlain"><?= _t('gen.auth.password') ?></label>
-			<input type="password" id="passwordPlain" required="required" />
+			<div class="stick">
+				<input type="password" id="passwordPlain" required="required" />
+				<a  class="btn toggle-password" data-toggle="passwordPlain"><?= _i('key') ?></a>
```
Comment (SuperSandro2000) [inline app/views/auth/formLogin.phtml]:
```suggestion
				<a class="btn toggle-password" data-toggle="passwordPlain"><?= _i('key') ?></a>
```

File: p/themes/Alternative-Dark/adark.css [hunk 2]
```diff
@@ -663,6 +663,12 @@ a.btn {
 
 /*=== Prompt (centered) */
 .prompt {
+	max-width: 20rem;
+	margin-left: auto;
+	margin-right: auto;
+	padding-left: .5rem;
+	padding-right: .5rem;
+
 	text-align: center;
```
Comment (SuperSandro2000) [inline p/themes/Alternative-Dark/adark.css]:
```suggestion
	padding-right: .5rem;
	text-align: center;
```

File: p/themes/Alternative-Dark/adark.css [hunk 3]
```diff
@@ -671,13 +677,39 @@ a.btn {
...
+
+.prompt .form-group::after {
+	display: none;
+}
+
+.prompt .form-group.form-group-actions {
+	display: flex;
+	margin-top: 2rem;
+
+	align-items: center;
```
Comment (SuperSandro2000) [inline p/themes/Alternative-Dark/adark.css]:
```suggestion
	margin-bottom: 3rem;
	text-align: left;
}

.prompt .form-group {
	margin-bottom: 1rem;
}

.prompt .form-group::after {
	display: none;
}

.prompt .form-group.form-group-actions {
	display: flex;
	margin-top: 2rem;
	align-items: center;
```

**Review by SuperSandro2000 — COMMENTED** [review]
You get the gist with my 3 suggestions. :)

**Comment by Alkarex** [comment]
Thanks @flo0627 , that looks good 👍 
And thanks @SuperSandro2000 for the reviewing help!

**Review by Alkarex — APPROVED** [review]

**Comment by Alkarex** [comment]
@flo0627 Please add a line for you in https://github.com/FreshRSS/FreshRSS/blob/master/CREDITS.md

**Pull request merged**